starters/nuxt3-pinia-vuetify/.storybook/preview.js

```diff
@@ -0,0 +1,56 @@
+import "vuetify/dist/vuetify.css";
+import { create } from '@storybook/theming';
+import { action } from '@storybook/addon-actions';
+import { app, addDecorator } from '@storybook/vue3';
+import { Vuetify } from '../plugins/vuetify';
+import { createPinia } from 'pinia';
+
+
+// Support for pina in Storybook
+const pinia = createPinia();
+app.use(pinia);
+
+// Support for vuetify in Storybook
+app.use(Vuetify);
```

Comment:
Perhaps we could follow this example from their docs: https://storybook.js.org/recipes/vuetify#register-vuetify-in-storybook